src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/MenuItem.tsx

```diff
@@ -1,20 +1,40 @@
 // Libraries
 import React, {FC} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
 import {
   AlignItems,
   Dropdown,
   FlexBox,
   Icon,
   IconFont,
 } from '@influxdata/clockface'
-import {useDispatch} from 'react-redux'
 
 // Components
 import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
 
+// Selectors
+import {
+  selectCurrentAccountId,
+  selectCurrentAccountType,
+  selectUser,
+} from 'src/identity/selectors'
+
+// Eventing
+import {HeaderNavEvent, multiOrgTag} from 'src/identity/events/multiOrgEvents'
+import {event} from 'src/cloud/utils/reporting'
+
 export const CreateOrganizationMenuItem: FC = () => {
   const dispatch = useDispatch()
+  const user = useSelector(selectUser)
+  const accountId = useSelector(selectCurrentAccountId)
+  const accountType = useSelector(selectCurrentAccountType)
+
   const handleCreateOrg = () => {
+    event(HeaderNavEvent.CreateOrgClick, multiOrgTag, {
+      userId: user.id,
+      accountId,
+      accountType,
+    })
```

Comment:
For this event and the next one, I might add `accountName` here, which will be easier for analytics if they're browsing the data later. That way they don't need to join SQL tables to see 'oh, this account refers to this company'.